### ISSUE TITLE
[Image Caption] Incorrect output from pre-trained demo

### ISSUE THREAD

**Issue by liyi14** [yunjey/pytorch-tutorial#167]
I followed the steps in README and tested the pre-trained network with the example.png, but get the following sentence:
`<start> sundown cuckoo deciding ages fastened company warning pouring active dim <end>`
Seems something is wrong? I am using python 3.6 with pytorch 1.0.
Can you give me some hint about what might be the problem?
Thanks for this great project!

**Comment by liyi14** [comment]
I noticed that the vocab.pkl was replaced when I compile the dataset. If I use the one from the download link, the result is correct.
